### DISCUSSION TITLE
[Feature] Partner share filter

### DISCUSSION THREAD

**Discussion by rgareh** [immich-app/immich#11361] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

It would be awesome if you could apply a filter to what is shared with partner. ie, filter by date (share everything from xx/xx/xxx onwards), or filter out certain people from the past.

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
#2995
